app-commons/src/main/scala/org/bitcoins/commons/serializers/JsonReaders.scala

```diff
@@ -843,6 +889,35 @@ object JsonReaders {
       }
   }
 
+  implicit val channelCommandStateReads: Reads[ChannelCommandState] = Reads {
+    jsValue =>
+      SerializerUtil.processJsString { s =>
+        if (s == "ok") {
+          OK
+        } else if (s.startsWith("created channel ")) {
+          ChannelOpened
+        } else if (s.startsWith("closed channel ")) {
```

Comment:
Are these supposed end with a `" "`?

Reply:
Also, would be nice to just have a `ChannelCommandState.fromString`

Reply:
> Are these supposed end with a `" "`?

Yes